dev/ci/render-pr-preview.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+echo "--- Render.com PR Preview"
+
+curl -sS --request POST \
```

Comment:
```suggestion
curl -sSf --request POST \
```

( `-f` is to make curl fail early in case of non OK http response)